packages/react-native-bpk-appearance/src/hooks-flowtest.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable backpack/use-tokens */
+/* @flow */
+import { useBpkDynamicValue, useBpkDynamicStyle } from './hooks';
+import { type BpkDynamicValue } from './common-types';
```

Comment:
This file is only here to ensure most cases continue to work as we change the flow config, because it is very hard to know if all possible cases are working otherwise

Reply:
Given that most of the lines here are `eslint-disable-line` would it make sense to just do `eslint-disable` for the whole file?